lib/init-7.scm

```diff
@@ -873,8 +873,8 @@
       (set! count (+ count 1))
       (rename (string->symbol (string-append s (%number->string count)))))
     (define (expand-pattern pat tmpl)
-      (let lp ((p (cdr pat))
-               (x (list _cdr _expr))
+      (let lp ((p pat)
+               (x _expr)
```

Comment:
Did you forget to revert this?

Reply:
I haven’t changed anything since you pushed your proposed change.